catalog/ui/catalog-ui-search/src/main/webapp/component/workspace-details/workspace-details.hbs

```diff
@@ -19,5 +19,12 @@
     {{bind selector="> .choice-date" event="change:modified" key=(path "niceDate")}}
 </div>
 <div class="choice-owner" title="{{bindAttr attr="title" selector="> .choice-owner" event="change:metacard.owner" key=(path "owner")}}" data-help="The owner of the workspace.">
-    {{bind selector="> .choice-owner" event="change:metacard.owner" key=(path "owner")}}
+    {{#if localStorage}} 
```

Comment:
Just curious, is there a reason this `if` is applied on the whole `<span>` instead of just the `fa-home`/`fa-cloud` classes?